isis/src/base/objs/SurfacePoint/SurfacePoint.cpp

```diff
@@ -924,14 +895,10 @@ namespace Isis {
         Angle latSigma = GetLatSigma();
 
         if (latSigma.isValid()) {
-          if (!p_majorAxis || !p_majorAxis->isValid()) {
-            IString msg = "In order to calculate sigmas in meter units, the "
-              "equitorial radius must be set with a call to SetRadii.";
-            throw IException(IException::Programmer, msg, _FILEINFO_);
-          }
+          Distance scalingRadius = GetLocalRadius();
```

Comment:
This local variable can be removed and line 901 can just be `latSigmaDistance = latSigma.radians() * GetLocalRadius();`